src/credits.c

```diff
@@ -128,16 +128,17 @@ enum
     TDE_TASK_A_ID = 2,
 };
 
-struct Unk201C000
-{
-    u16 unk0[71];
-    u16 unk8E;
-    u16 unk90;
-    u16 unk92;
-    u16 unk94;
-    u16 unk96[NATIONAL_DEX_COUNT];
-    u16 unk39A;
-    u16 unk39C[7];
+#define NUM_PKMN_SLIDES 71
```

Comment:
Please use `mon` instead of `pkmn` when abbreviating "pokemon" to be consistent with the rest of the codebase.